guide/english/javascript/index.md

```diff
@@ -15,20 +15,30 @@ If you want to learn more about the JavaScript language, and why it's so widely
 or watch this [inspiring video from Preethi Kasireddy](https://www.youtube.com/watch?v=VqiEhZYmvKk).
 
 ## Standalone JavaScript engines
-Mozilla's SpiderMonkey, the first JavaScript engine ever written, currently used in Mozilla Firefox.
+-Mozilla's SpiderMonkey, the first JavaScript engine ever written, currently used in Mozilla Firefox.
```

Comment:
All the list items are not properly formatted according to Markdown as previously they were. Please format them correctly.